tests/test_machine.py

```diff
@@ -1,69 +1,82 @@
 # SPDX-License-Identifier: LGPL-2.1+
 
-import os
-from subprocess import TimeoutExpired
-
+import unittest
 import pytest
+from subprocess import TimeoutExpired
+from sys import exc_info
+from typing import List, Tuple, Optional, Type
+from types import TracebackType
 
-import mkosi.machine as machine
 from mkosi.backend import MkosiException
-
-
-pytestmark = [
-    pytest.mark.integration,
-    pytest.mark.parametrize("verb", ["boot", "qemu"]),
-    pytest.mark.skipif(os.getuid() != 0, reason="Must be invoked as root.")
-]
-
-
-def test_simple_run(verb: str) -> None:
-    with machine.Machine([verb]) as m:
-        p = m.run(["echo", "This is a test."])
-        assert "This is a test." == p.stdout.strip("\n")
-
-    assert m.exit_code == 0
-
-
-def test_wrong_command(verb: str) -> None:
-    # First tests with argument check = True from mkosi.backend.run(), therefore we see if an exception is raised
-    with machine.Machine([verb]) as m:
+from mkosi.machine import Machine
+
+
+class MkosiMachineTest(unittest.TestCase):
```

Comment:
We should not put this class in test_machine as we want it to be reusable across multiple test files. You can put this class next to the `Machine` class in mkosi/machine.py